front/src/WebRtc/SimplePeer.ts

```diff
@@ -145,7 +146,11 @@ export class SimplePeer {
         }
 
         mediaManager.removeActiveVideo("" + user.userId);
-        mediaManager.addActiveVideo("" + user.userId, name);
+
+        //TODO ad condition isPublic or annonyme
```

Comment:
Waiting for the condition here before merging this PR (otherwise, existing users will notice)